src/scripts/components/stories/story-gallery/story-gallery.tsx

```diff
@@ -3,11 +3,12 @@ import cx from 'classnames';
 
 import {PreviousIcon} from '../../main/icons/previous-icon';
 import {NextIcon} from '../../main/icons/next-icon';
-import {FullscreenExitIcon} from '../../main/icons/fullscreen-exit-icon';
 import {FullscreenIcon} from '../../main/icons/fullscreen-icon';
 import {getStoryAssetUrl} from '../../../libs/get-story-asset-urls';
 import {useInterval} from '../../../hooks/use-interval';
 import config from '../../../config/main';
+import Description from '../description/description';
```

Comment:
The name `Description` is to general here. How about `<Caption content="" />`?

Reply:
Good point, will change it.

Regarding this:
> Do we have to use text-align: left here. It looks weird with short texts on larger screens, I would prefer text-align: center

In the new designs the captions are all aligned left, but you're right it look weird with short text. I will adjust it.

Reply:
![image](https://user-images.githubusercontent.com/45818654/111166598-8a827880-85a0-11eb-818e-0b2c23da87fd.png)

![image](https://user-images.githubusercontent.com/45818654/111166665-99692b00-85a0-11eb-9b05-028d621fa07d.png)

Reply:
I changed the background and border of the dots so you can see the progress on a white background:

![image](https://user-images.githubusercontent.com/45818654/111169459-40e75d00-85a3-11eb-811a-b32feffa0622.png)

Reply:
nice!